### ISSUE TITLE
Missing coverage

### ISSUE THREAD

**Issue by zandaqo** [denoland/deno#11875]
`deno coverage` shows missing coverage for a number of methods in my library that are in fact being called in tests and were previously shown as covered when tested with Jest. To reproduce:
```bash
git clone https://github.com/zandaqo/structurae.git
cd structurae
deno test --coverage=cov
deno coverage cov
```
I cannot yet pinpoint the commonalities between skipped methods, but there are certain patterns:
1. `[Symbol.species]()` accessor calls are missing in most cases even when being accessed directly in tests, although with Jest they would be covered when called indirectly with `slice`, `splice`, etc. e.g:
```typescript
// code https://github.com/zandaqo/structurae/blob/master/adjacency-list.ts#L35
    static get [Symbol.species]() {
      return Base;
    }

// test 
test("[AdjacencyList.$species] returns underlying TypedArray when sliced", () => {
  assertEquals(DirectedList[Symbol.species], Uint32Array);
});
```
2. `*[Symbol.iterator]()` is also missing in most cases:
```typescript
// code https://github.com/zandaqo/structurae/blob/master/array-view.ts#L74
  *[Symbol.iterator](): Iterator<ViewInstance<T>> {
    const { size } = this;
    for (let i = 0; i < size; i++) {
      yield this.getView(i);
    }
  }

// test 
test("[ArrayView#[Symbol.iterator]] iterates over elements of the array", () => {
  const arrayView = Uint32ArrayView.from([30, 40, 50]);
  const array = [...arrayView];
  assertEquals(array[0] instanceof Uint32View, true);
  assertEquals(array[0].get(), 30);
  assertEquals(array.length, 3);
});
```

I would attribute these to using the symbols in mixins, but there are also cases where the same pattern is being covered, e.g. [Grid](https://github.com/zandaqo/structurae/blob/045ccf0c6750c14543246116a33dc5b83ff91107/grid.ts#L21).

**Comment by bartlomieju** [comment]
I'm able to reproduce the problem with `Symbol` methods not being covered. I'll try to boil down the example to two-file reproductions and need to verify if it's problem with applying source maps or coverage data is actually not collected by V8.

**Comment by bartlomieju** [comment]
I'm 95% confident this is directly related to https://github.com/denoland/deno/issues/10944 and should be fixed in #13334
